locales/sv_SE/pcsx2_Main.po

```diff
@@ -1011,14 +1009,14 @@ msgstr "32MB fil"
 msgid "64MB File"
 msgstr "64MB fil"
 
-#: pcsx2/gui/Dialogs/ConvertMemoryCardDialog.cpp:95
-msgid "Convert this memory card to a folder of individual saves."
-msgstr "Omvandla detta Minneskort till en mapp med individuella sparningar."
-
 #: pcsx2/gui/Dialogs/ConvertMemoryCardDialog.cpp:95
 msgid "Folder"
 msgstr "Mapp"
 
+#: pcsx2/gui/Dialogs/ConvertMemoryCardDialog.cpp:95
+msgid "Convert this memory card to a folder of individual saves."
+msgstr "Omvandla detta Minneskort till en mapp med individuella sparningar."
```

Comment:
```suggestion
msgstr "Omvandla detta minneskort till en mapp med individuella sparningar."
```

Reply:
I like "Minneskort" over "minneskort" since it makes it more readable.